File: src/main/java/com/vspk/bookrest/dto/AuthUserDetailsDto.java
package com.vspk.bookrest.dto;

import lombok.Data;

import javax.validation.constraints.NotBlank;

@Data
public class AuthUserDetailsDto {
    @NotBlank(message = "username should not be empty")
    private String username;
    @NotBlank(message = "password should not be empty")
    private String password;
}

File: src/main/java/com/vspk/bookrest/dto/AuthUserDetailsDto.kt
package com.vspk.bookrest.dto

import javax.validation.constraints.NotBlank
import lombok.Data


data class AuthUserDetailsDto(
    val username: @NotBlank(message = "username should not be empty") String,
    val password: @NotBlank(message = "password should not be empty") String
)
